contrib/terraform/upcloud/modules/kubernetes-cluster/main.tf

```diff
@@ -531,6 +534,10 @@ resource "upcloud_loadbalancer_frontend" "lb_frontend" {
   mode                 = "tcp"
   port                 = each.value.port
   default_backend_name = upcloud_loadbalancer_backend.lb_backend[each.key].name
+  properties {
+    inbound_proxy_protocol = var.loadbalancer_inbound_proxy_protocol
```

Comment:
Isnt this the listener? Would we want proxy protocol there? If it is I think that we would at least want to have it separatable so you can enable them individually

Reply:
I made it like this since you need both for it to work

![image](https://github.com/elastisys/kubespray/assets/22236722/2415d417-895a-4d2d-893a-0339b62d161a)

Although I am not sure that it works anyways, I wasn't able to get it to work, so I will open an issue to investigate this.